### PR TITLE
Added MODX

### PR DESCRIPTION
**Pull request by vierkantemeter**
In conjunction with PR https://github.com/sendgrid/docs/pull/1561

### CODE REVIEW

File: source/Integrate/Open_Source_Apps/index.html [hunk 1]
```diff
@@ -38,5 +38,11 @@
         <span class="icon-wordpress"></span>
       </a>
     </li>
+       <li>
+      <a href="modx.html">
+        MODX
+        <span class="icon-modx"></span>
```
Comment (mbernier) [inline source/Integrate/Open_Source_Apps/index.html]: I don't believe we have this icon...

Comment (mbernier) [inline source/Integrate/Open_Source_Apps/index.html]: If you have it, could you send it over?

Comment (vierkantemeter) [inline source/Integrate/Open_Source_Apps/index.html]: I saw you are using the icomoon fontpack, which doesn't have the MODX logo included. The FontAwesome pack does.

I only have the logo as an transparent PNG:

modx

Comment (mbernier) [inline source/Integrate/Open_Source_Apps/index.html]: @katieporter would you be able to help us add this icon in?

**Pull request merged**